### PR TITLE
Add UIColor extension according to the new colors guideline

### PR DESCRIPTION
**Pull request by mazevedofs**
**Phabricator:** https://phabricator.wikimedia.org/T299793

### Notes
*  Adds the new app colors

### CODE REVIEW

File: Sources/Components/Colors.swift [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import UIKit
+
+public extension UIColor {
+
+	static let gray800 = UIColor(0x101418)
```
Comment (mazevedofs) [inline Sources/Components/Colors.swift]: The indentation on this file seems wrong, but on Xcode it looks right

File: Sources/Components/Colors.swift [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+	static let gray700 = UIColor(0x202122)
+	static let gray675 = UIColor(0x27292D)
+	static let gray650 = UIColor(0x2E3136)
+	static let gray600 = UIColor(0x54595D)
+	static let gray500 = UIColor(0x72777D)
+	static let gray450 = UIColor(0x8E8E93)
```
Comment (tonisevener) [inline Sources/Components/Colors.swift]: I don't think this color is used anywhere.

File: Sources/Components/Colors.swift [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+	static let gray150 = UIColor(0xEEF2FB)
```
Comment (tonisevener) [inline Sources/Components/Colors.swift]: Very minor - can you move gray150 to be after gray200?

**Review by tonisevener — CHANGES_REQUESTED** [review]

File: Sources/Components/Colors.swift (hunk 1 above)
Comment (tonisevener) [reply Sources/Components/Colors.swift]: May have something to do with your Xcode indention settings for this project. Mine are set to prefer indent using spaces, tab and indent width both set to 4.

Comment (mazevedofs) [reply Sources/Components/Colors.swift]: Mine was set to tabs, that was it!

File: Sources/Components/Colors.swift (hunk 3 above)
Comment (mazevedofs) [reply Sources/Components/Colors.swift]: Done!

File: Sources/Components/Colors.swift (hunk 2 above)
Comment (mazevedofs) [reply Sources/Components/Colors.swift]: I removed it, we can add it back if it is ever necessary

Comment (tonisevener) [reply Sources/Components/Colors.swift]: I don't see the change, might need a push?

File: Sources/Components/Colors.swift (hunk 3 above)
Comment (tonisevener) [reply Sources/Components/Colors.swift]: I don't see the change, might need a push?

Comment (mazevedofs) [reply Sources/Components/Colors.swift]: Yes 🤦‍♀️

**Review by tonisevener — APPROVED** [review]

**Pull request merged**